packages/components/src/components/post-card-control/post-card-control.tsx

```diff
@@ -148,11 +148,31 @@ export class PostCardControl {
     checkNonEmpty(this, 'label');
   }
 
+  @Watch('name')
+  validateControlName() {
+    checkNonEmpty(this, 'name', undefined, this.type === 'radio' ? 'error' : 'warning');
+  }
+
+  @Watch('value')
+  validateControlValue() {
+    checkNonEmpty(this, 'value', undefined, this.type === 'radio' ? 'error' : 'warning');
+  }
+
+  @Watch('description')
+  validateControlDescription() {
+    checkNonEmpty(this, 'description', undefined, 'warning');
+  }
+
```

Comment:
Can you check all the properties you validated to know if they are actually mandatory or not, and adapt either the validator or the declaration? 

To know if a property is mandatory you have to check how it is used: 
- if it is used within `if` conditions then it is probably optional, declared with `?:` and validated with a `checkEmptyOr` validator,
- if it is used without a condition then it is mandatory, declared with `!:` and validated with the `checkNonEmpty()` validator.